stream-chat-android/src/main/java/com/getstream/sdk/chat/view/channels/ChannelListViewStyle.kt

```diff
@@ -6,44 +6,38 @@ import android.graphics.Typeface
 import android.util.AttributeSet
 import androidx.annotation.ColorRes
 import androidx.annotation.LayoutRes
+import androidx.core.content.ContextCompat
 import com.getstream.sdk.chat.R
 import com.getstream.sdk.chat.style.TextStyle
-import com.getstream.sdk.chat.view.BaseStyle
+import com.getstream.sdk.chat.view.ReadStateStyle
+import com.getstream.sdk.chat.view.messages.AvatarStyle
 
-internal class ChannelListViewStyle(context: Context, attrs: AttributeSet?) : BaseStyle() {
-
-    @JvmField
+public class ChannelListViewStyle(context: Context, attrs: AttributeSet?) {
     @LayoutRes
-    var channelPreviewLayout: Int
-
-    @JvmField
-    val channelTitleText: TextStyle
-
-    @JvmField
-    val channelTitleUnreadText: TextStyle
-
-    @JvmField
-    val lastMessage: TextStyle
-
-    @JvmField
-    val lastMessageUnread: TextStyle
-
-    @JvmField
-    val lastMessageDateText: TextStyle
-
-    @JvmField
-    val lastMessageDateUnreadText: TextStyle
+    public val channelPreviewLayout: Int
+    public val channelTitleText: TextStyle
+    public val channelTitleUnreadText: TextStyle
+    public val lastMessage: TextStyle
+    public val lastMessageUnread: TextStyle
+    public val lastMessageDateText: TextStyle
+    public val lastMessageDateUnreadText: TextStyle
+    public var avatarStyle: AvatarStyle = AvatarStyle()
+    public val readStateStyle: ReadStateStyle = ReadStateStyle()
```

Comment:
Looks to me like these should be `lateinit` and `private set`, or maybe just a `val` if we can refactor things around them a bit.